pkgs/test_core/lib/src/runner/engine.dart

```diff
@@ -293,6 +302,30 @@ class Engine {
     return success;
   }
 
+  Future<Null> _gatherCoverage(LiveSuiteController controller) async {
+    if (_coverage == null) return;
+
+    final RunnerSuite suite = controller.liveSuite.suite;
+
+    if (!suite.platform.runtime.isDartVM) return;
+
+    // for some reason the observatoryUrl URI doesn't have the query parameters populated
```

Comment:
I think this comment is outdated now that we've figured out it comes from the fragment, right?

Reply:
Fixed!